handlers/hormuud/handler.go

```diff
@@ -79,7 +79,7 @@ type mtPayload struct {
 func (h *handler) Send(ctx context.Context, msg courier.MsgOut, res *courier.SendResult, clog *courier.ChannelLog) error {
 	token, err := h.FetchToken(ctx, msg.Channel(), msg, clog)
 	if err != nil {
-		return errors.Wrapf(err, "unable to fetch token")
+		return errors.New("unable to fetch token")
```

Comment:
why are you getting rid of the wrapping? I still don't undertstand why this isn't `fmt.Errorf("unable to fetch token: %w", err)` and why you need to change the tests....

Reply:
```
actual  : *fmt.wrapError(&fmt.wrapError{msg:"unable to fetch token: error making token request: %!w(<nil>)", err:(*fmt.wrapError)(0x14000796100)})
```

Reply:
That is what we get when we run the test, how do we write the test to match that?

Reply:
the problem is we have an existing bug...

```go
if err != nil || resp.StatusCode/100 != 2 {
	return "", errors.Wrapf(err, "error making token request")
}
```

if `err == nil` but `resp.StatusCode` is non-200... that returns `"", nil`